samples/system-test/v3beta1/translate_batch_translate_text_beta.test.js

```diff
@@ -0,0 +1,58 @@
+/**
+ * Copyright 2019, Google, Inc.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+'use strict';
+
+const {assert} = require('chai');
+const {TranslationServiceClient} = require('@google-cloud/translate').v3beta1;
+const {Storage} = require('@google-cloud/storage');
+const execa = require('execa');
+const uuid = require('uuid');
+const exec = async cmd => (await execa.shell(cmd)).stdout;
+
+const REGION_TAG = 'translate_batch_translate_text_beta';
+
+describe(REGION_TAG, () => {
+  const translationClient = new TranslationServiceClient();
+  const location = `us-central1`;
```

Comment:
prefer single-quotes over tick (`) for non-template literals

Reply:
Apologies, folks shouldn't have to guess.  Filed https://github.com/googleapis/google-cloud-node/issues/2915 to make sure the linter/fixer catches this in the future.